gptel.el

```diff
@@ -240,9 +240,14 @@ all at once.  This wait is asynchronous.
   :type 'boolean)
 (make-obsolete-variable 'gptel-playback 'gptel-stream "0.3.0")
 
-(defcustom gptel-use-curl (and (executable-find "curl") t)
```

Comment:
Why rename this defcustom?  If you have multiple versions of curl on your path, you can set Emacs' process environment appropriately to pick the right one.

Reply:
I went off the discussion at https://github.com/karthink/gptel/pull/145

My gut reaction as someone whose platform curl doesn't support AWS signatures is that messing with process path is kind of painful to do. I would probably just put an updated `curl` in `~/.local/bin` and accept that curl is de facto no longer managed by my distro, but that seems kind of icky